### ISSUE TITLE
Disallow belonging to multiple categories?

### ISSUE THREAD

**Issue by paulirish** [patrickhulce/third-party-web#194]
Only 6 out of all 2099 entities have more than 1 category. Here's the list:

```js
[{
    name: 'KARTE',
    homepage: 'https://karte.io/',
    categories: [ 'analytics', 'marketing' ]
  }, {
    name: 'LoginRadius',
    homepage: 'https://www.loginradius.com/',
    categories: [ 'ad', 'social' ]
  }, {
    name: 'Rtoaster',
    homepage: 'https://www.brainpad.co.jp/rtoaster/',
    categories: [ 'analytics', 'marketing' ]
  }, {
    name: 'SalesLoft',
    homepage: 'https://salesloft.com/',
    categories: [ 'analytics', 'marketing' ]
  }, {
    name: 'Pepperjam',
    homepage: 'https://www.pepperjam.com/',
    categories: [ 'marketing', 'analytics' ]
  }, {
    name: 'Segmanta',
    homepage: 'https://segmanta.com/',
    categories: [ 'marketing', 'analytics' ]
}]
```

As a consumer of this data, it'd be a lot better if I could ensure an entity doesn't show up in multiple. And since we're so close... Do folks think it'd be possible to be decisive and settle these 6? We could then assert categories.length === 1 moving forward

wdyt?

**Comment by eligrey** [comment]
I vote against this suggestion. Companies are often known to bundle multiple discrete services with different tracking purposes on the same domain.

What would you categorize google.com?

**Comment by patrickhulce** [comment]
Heya Paul :) Yeah I can get behind that given our lack of bandwidth to do more fine-grained categorization, but I'd prefer to relabel `category` than have a hidden invariant. Should we want to add multiple categories again, I think it's best to segment by product anyhow rather than blindly saying the whole entity is both.

@eligrey you point is well taken but the reality is we've largely avoided such multi-labels (as @paulirish notes there are only 6) and haven't had the resources to do significant fine grained labeling.

**Comment by patrickhulce** [comment]
fixed by #195
